File: tests/C/0003.c
int main (int argc, char *argv[])
{
	unsigned N = 1000;

	#pragma acc parallel loop
	for (unsigned i = 0; i < N; ++i)
	{
	}

	#pragma \
	  acc parallel loop
	for (unsigned i = 0; i < N; ++i)
	{
	}

	  #pragma \
	    acc parallel loop
	for (unsigned i = 0; i < N; ++i)
	{
	}

	  #pragma \
	    acc \
	    parallel loop
	for (unsigned i = 0; i < N; ++i)
	{
	}

	  #pragma \
	    acc \
	    parallel \
	    loop
	for (unsigned i = 0; i < N; ++i)
	{
	}
	return 0;
}

File: tests/specific-tests/preprocessor-guards/0003.f90
program test
integer N
integer i

!$acc parallel &
!$acc & loop
      do i = 1, N
      enddo

!$acc parallel &
!$acc loop
      do i = 1, N
      enddo

!$acc parallel &
!$acc loop
      do i = 1, N
      enddo

!$acc parallel &
!$acc     & loop
      do i = 1, N
      enddo
end program
